src/pseudopeople/loader.py

```diff
@@ -52,49 +53,59 @@ def load_and_prep_1040_data(data_path: dict, user_filters: List[Tuple]) -> pd.Da
     )
     # Rename tax_dependents columns
     dependents_wide = dependents_wide.add_prefix("dependent_").reset_index()
+    # Make sure we have all dependent columns if data does not have a guardian with 4 dependents
+    if COLUMNS.dependent_4_first_name.name not in dependents_wide.columns:
```

Comment:
So this updates any missing dependent_4 columns w/ np.nan. But what about dependents 2-3?

Reply:
That's a good point. This was added because our sample data didnt have a guaridan with 4 dependents but verifying the correct schema/columns should be more robust.